src/xcode/ENA/ENA/Source/Scenes/Home/HomeInteractor.swift

```diff
@@ -229,7 +229,7 @@ extension HomeInteractor {
 		riskLevelConfigurator = nil
 		inactiveConfigurator = nil
 
-		let detectionInterval = (riskProvider.configuration.exposureDetectionInterval.day ?? 1) * 24
+        let detectionInterval = riskProvider.riskProvidingConfiguration.exposureDetectionInterval.hour ?? RiskProvidingConfiguration.defaultExposureDetectionsInterval
```

Comment:
Maybe too niggling 😸  Just a missing inline tab (code styling)

Reply:
No problem. But it looks ok ob my side. No missing tab 😇
I tried to fix it by removing and adding the tab again 😅

Reply:
Nobody likes tabs. We should use white spaces 🙃
Github has issues with displaying this. We can mostly ignore it. I would bet `swiftlint` will find any misaligned code.